bundles/framework/admin-announcements/view/AnnouncementsFormMapperHandler.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { StateHandler, controllerMixin, Messaging } from 'oskari-ui/util';
+import { announcementsCalls } from './AnnouncementsCalls';
+import { Message } from 'oskari-ui';
+
+/*
+Handler for admin-announcements forms.
+*/
+
+const getMessage = (key, args) => <Message messageKey={key} messageArgs={args} bundleKey='admin-announcements' />;
+
+class ViewHandler extends StateHandler {
+    constructor (instance) {
```

Comment:
We only need instance to get access to localization and sandbox here. You should use those as params instead of instance. But it also looks like you don't need those here at all or without changes to delete confirmation you need the localization at most. Change the parameters here accordingly. Pref refactor to use no params or just send localization or even the error message since that is all you need it for.